### ISSUE TITLE
Cannot login after server update

### ISSUE THREAD

**Issue by amiguet** [bain3/pronotepy#284]
I've got a script to automate some tasks that has been working for about one year, but the school updated pronote and now I cannot login any more:

```python
>>> client = pronotepy.ParentClient('https://.../parent.html',
    username='...'
    password='...'
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File ".../venv/lib/python3.11/site-packages/pronotepy/clients.py", line 643, in __init__
    super().__init__(pronote_url, username, password, ent, qr_code)
  File ".../venv/lib/python3.11/site-packages/pronotepy/clients.py", line 352, in __init__
    super().__init__(pronote_url, username, password, ent, qr_code)
  File ".../venv/lib/python3.11/site-packages/pronotepy/clients.py", line 75, in __init__
    self.attributes, self.func_options = self.communication.initialise()
                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".../venv/lib/python3.11/site-packages/pronotepy/pronoteAPI.py", line 100, in initialise
    initial_response = self.post(
                       ^^^^^^^^^^
  File ".../venv/lib/python3.11/site-packages/pronotepy/pronoteAPI.py", line 181, in post
    raise PronoteAPIError(
pronotepy.exceptions.PronoteAPIError: Unknown error from pronote: 21 | Impossible d'afficher la page
```
Any idea what's happening?

**Comment by bain3** [comment]
Hi,
do you have the latest version installed?

**Comment by amiguet** [comment]
I'm using pronotepy 2.12.1 installed in a fresh venv with `pip install pronotepy`

`pip freeze` yields

```
autoslot==2022.12.1
beautifulsoup4==4.12.2
certifi==2023.7.22
charset-normalizer==3.3.0
idna==3.4
pronotepy==2.12.1
pycryptodome==3.19.0
requests==2.31.0
soupsieve==2.5
urllib3==2.0.7
```

**Comment by amiguet** [comment]
As I couldn't find how to log in the old way, I tried the QR-code based method described here: https://pronotepy.readthedocs.io/en/stable/quickstart.html#usage

But, no matter if I use method 1 (QR) or 2 (user/pass), `python3 -m pronotepy.create_login` yields the same error as above:

```
pronotepy.exceptions.PronoteAPIError: Unknown error from pronote: 21 | Impossible d'afficher la page
```

So I'm stuck, I've got no way to login whatsoever. 

Any idea?

**Comment by bain3** [comment]
Honestly no idea at all... Could you share a link to your PRONOTE instance?

If you're not comfortable with sharing it publicly, maybe you could send it by email (or any other means listed on [bain.cz](https://bain.cz/))?

**Comment by bain3** [comment]
Should be fixed in 675b954. amiguet's PRONOTE had encryption enabled even though the connection was over https (sigh).

**Comment by amiguet** [comment]
Works perfectly, many thanks! :-)
